lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -81,6 +83,36 @@ class DetailsRenderer {
     element.appendChild(items);
     return element;
   }
+
+  /**
+   * @param {!DetailsJSON} list
+   * @return {!Element}
+   */
+  _renderCards(list) {
+    const element = this._dom.createElement('details', 'lighthouse-details');
+    if (list.header) {
+      element.appendChild(this._dom.createElement('summary')).textContent = list.header.text;
```

Comment:
why are we keeping the verbose `header: {type: 'text', text: 'View details'}` format if we don't do anything with it (here or in the list formatter)?